Update README.md to have ShutterFeed name

CamilBouzidi opened the pull request:
Removed the references to the redacted name and replaced them with our new ShutterFeed name. I will be requiring all of your approvals, just to ensure that no one dislikes the name!

CamilBouzidi committed:
Update README.md to have ShutterFeed name

Removed the references to the redacted name and replaced them with our new ShutterFeed name.

laurent-lao reviewed (changes requested):
Can you remove the dashes that wrap the name?

CamilBouzidi committed:
Removed wrapping hyphens

per @laurentlaurent 's suggestion, I have removed the hyphens that were wrapping the name.

laurent-lao reviewed (approved):
Looks good to me
